src/main/kotlin/org/rust/ide/actions/RustfmtFileAction.kt

```diff
@@ -26,15 +37,26 @@ class RustfmtFileAction : DumbAwareAction() {
     override fun actionPerformed(e: AnActionEvent) {
         val (project, toolchain, file) = getContext(e) ?: return
 
-        val cargo = toolchain.rawCargo()
+        FileDocumentManager.getInstance().saveAllDocuments()
+
+        val rustfmt = toolchain.rustfmt()
         try {
-            cargo.reformatFile(project, file)
+            ProgressManager.getInstance().runProcessWithProgressSynchronously<ProcessOutput, ExecutionException>({
+                rustfmt.reformatFile(project, file)
+            }, "Reformat File with Rustfmt", true, project)
```

Comment:
The title describes a process that is currently running, so it should be in continuous form and with ellipsis in the end: "Reformatting File with Rustfmt..."
![image](https://user-images.githubusercontent.com/3221931/44032725-a9635644-9f10-11e8-8967-4de130dd312d.png)